docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+services:
```

Comment:
this might be a good opportunity to remove the postgres and redis dependencies from the core sourcebot image? or are you planning on doing that later since its a breaking change